### DISCUSSION TITLE
[Basic 5000] vits japanese model

### DISCUSSION THREAD

**Discussion by neurlang** [coqui-ai/TTS#1604] [Show and tell]
マッチ試験に備えて、本当に気合を入れて勉強しなきゃ。 
Matchi shiken ni sonaete, hontōni kiai o irete benkyō shinakya.
mɑkki ʃikɛn ni sɔnɑɛtɛ, hɔntɔːni kiɑi ɔ irɛtɛ bɛnkjɔː ʃinɑkjɑ.

https://user-images.githubusercontent.com/77860779/170623826-7b690c67-6b5c-4fe4-b51f-589ce6ecc072.mp4

 カットの新決議案はかなりの衝撃を与える可能性もあります。 
Katto no shin ketsugi-an wa kanari no shōgeki o ataeru kanōsei mo arimasu.
gɑttɔ nɔ ʃinkɛtsugi ɑn vɑ, kɑnɑri nɔ ʃɔːgɛki ɔ, ɑtɑɛru kɑnɔːsɛː gɑ ɑrimɑsu

https://user-images.githubusercontent.com/77860779/170623830-83fee4bf-7807-4bc5-9c18-8b127c6e100d.mp4

**Comment by erogol** [comment]
Sounds good! Is it the end result or still training?

**Comment by neurlang** [comment]
this was step 60000, about 6 hour wallclock time.
it's sort of end result, although I now realized the ipa transcript still have mistakes in it. For instance, there is an ordinari ni used for に, there should be the IPA equivalent of ňi, as that's how the Japanese pronounce this.
The model learned to compenate for this, so when your input also have the same mistake, it speaks fine.

**Comment by neurlang** [comment]
#月には叢雲　華には風と
#朧深に隠れた　焦燥

tts --text="tsuki ni vɑ murɑkumɔ hɑnɑ ni vɑ kɑzɛ tɔ. ɔbɔrɔmi ni kɑkurɛtɑ ʃɔusɔ." $enc --out_path tts-poem-1.wɑv
ffmpeg -y -i tts-poem-1.wɑv tts-poem-1.mp4


https://user-images.githubusercontent.com/77860779/171323448-3bab5679-fe78-44b3-b1b0-c14257740c2a.mp4




#瞼焼き付いた顔
#理解者の証さえ

tts --text="mɑbutɑ jɑkitsuitɑ kɑvɔ. rikɑiʃɑ nɔ ɑkɑʃi sɑɛ." $enc --out_path tts-poem-2.wɑv
ffmpeg -y -i tts-poem-2.wɑv tts-poem-2.mp4



https://user-images.githubusercontent.com/77860779/171323460-be7ac860-ff06-41c8-bd6f-667dbd804545.mp4



#刹那、退屈の隙間贖い　心燻り
#不安を産み出した

tts --text="sɛtsunɑ, tɑikutsu nɔ sukimɑ ɑgɑnɑi kɔkɔrɔ kusuburi. huɑn vɔ umidɑʃitɑ." $enc --out_path tts-poem-3.wɑv
ffmpeg -y -i tts-poem-3.wɑv tts-poem-3.mp4



https://user-images.githubusercontent.com/77860779/171323481-da94860d-aeb6-431b-b48c-f41c76765a4a.mp4



#盲目消えた安らぎに出会って
#芽生えた恋情　譲る気は無い

tts --text="mɔːmɔku kiɛtɑ jɑsurɑgi ni dɛɑttɛ. mɛbɑɛtɑ rɛndʒɔu juzuru ki vɑ nɑi." $enc --out_path tts-poem-4.wɑv
ffmpeg -y -i tts-poem-4.wɑv tts-poem-4.mp4



https://user-images.githubusercontent.com/77860779/171323499-ef83dd01-c437-458a-a8a8-3116e0490db9.mp4




#月には叢雲　華には風と
#此方より彼方へ永久　築けぬなら
#雲突き抜け　風斬り裂いて
#久遠の揺蕩いへ　誘う

tts --text="tsuki ni vɑ murɑkumɔ hɑnɑ ni vɑ kɑzɛ tɔ. kɔnɑtɑ jɔri kɑnɑtɑ ɛ ɛikjuː kizukɛnu nɑrɑ." $enc --out_path tts-poem-5.wɑv
ffmpeg -y -i tts-poem-5.wɑv tts-poem-5.mp4


https://user-images.githubusercontent.com/77860779/171323567-0ff4554e-7b42-4184-8515-34632b394560.mp4




tts --text="kumɔ tsukinukɛ kɑzɛ kirisɑitɛ. kuɔn nɔ tɑjutɑi ɛ izɑnɑu." $enc --out_path tts-poem-6.wɑv
ffmpeg -y -i tts-poem-6.wɑv tts-poem-6.mp4



https://user-images.githubusercontent.com/77860779/171323580-d3999434-5a92-4e79-83a9-867a1a89a70c.mp4



#指と指が絡まり
#孤独を枯らしてゆく

tts --text="jubi tɔ jubi gɑ kɑrɑmɑri. kɔdɔku ɔ kɑrɑʃita juku." $enc --out_path tts-poem-7.wɑv
ffmpeg -y -i tts-poem-7.wɑv tts-poem-7.mp4



https://user-images.githubusercontent.com/77860779/171323595-2256c7da-9b82-44ac-b990-40473109c8ad.mp4



#真摯ゆえの想いの偏りは
#互いの意思を　繋ぐ要にして

tts --text="ʃinʃi juɛ nɔ ɔmɔi nɔ kɑtɑjɔri vɑ. tɑgɑi nɔ iʃi ɔ tsunɑgu kɑnɑmɛ ni ʃitɛ." $enc --out_path tts-poem-8.wɑv
ffmpeg -y -i tts-poem-8.wɑv tts-poem-8.mp4



https://user-images.githubusercontent.com/77860779/171323604-373094a8-e78b-4d88-8224-a83e91c69106.mp4



#毒にも似たこの絆は
#僅かな終焉の予感じゃ　千切れはしない

tts --text="dɔku ni mɔ nitɑ kɔnɔ kizunɑ vɑ. vɑzukɑ nɑ ʃuːɛn nɔ jɔkɑn dʒɑ tʃigirɛ vɑ ʃinɑi." $enc --out_path tts-poem-9.wɑv
ffmpeg -y -i tts-poem-9.wɑv tts-poem-9.mp4



https://user-images.githubusercontent.com/77860779/171323614-e839e242-815f-47a8-a2af-97e6a95d2405.mp4



#月には叢雲　華には風と
#たとえ愛の唄が永久　届かぬとも
#待ち続けよう　輪廻の時を
#慈しむ心に　誓って

tts --text="tsuki ni vɑ murɑkumɔ hɑnɑ ni vɑ kɑzɛ tɔ. tɑtɔɛ ɑi nɔ utɑ gɑ ɛikjuː tɔdɔkɑnu tɔmɔ." $enc --out_path tts-poem-10.wɑv
ffmpeg -y -i tts-poem-10.wɑv tts-poem-10.mp4


https://user-images.githubusercontent.com/77860779/171323639-fd51b3a1-dd43-43a0-be09-edb0715e9cc6.mp4



tts --text="mɑtʃi tsuzukɛjɔu rinnɛ nɔ tɔki vɔ. itsukuʃimu kɔkɔrɔ ni tʃikɑttɛ." $enc --out_path tts-poem-11.wɑv
ffmpeg -y -i tts-poem-11.wɑv tts-poem-11.mp4


https://user-images.githubusercontent.com/77860779/171323650-c6e1029d-4212-4af0-a782-1532deac4b51.mp4




#盲目消えた安らぎに出会って
#芽生えた恋情　譲る気は無い

tts --text="mɔːmɔku kiɛtɑ jɑsurɑgi ni dɛɑttɛ. mɛbɑɛtɑ rɛndʒɔu juzuru ki vɑ nɑi." $enc --out_path tts-poem-12.wɑv
ffmpeg -y -i tts-poem-12.wɑv tts-poem-12.mp4


https://user-images.githubusercontent.com/77860779/171323664-3237af93-a36d-4e83-8336-aceb773e0444.mp4



#月には叢雲　華には風と
#此方より彼方へ永久　築けぬなら
#雲突き抜け　風斬り裂いて
#久遠の揺蕩いへ　誘う

tts --text="tsuki ni vɑ murɑkumɔ hɑnɑ ni vɑ kɑzɛ tɔ. kɔnɑtɑ jɔri kɑnɑtɑ ɛ ɛikjuː kizukɛnu nɑrɑ." $enc --out_path tts-poem-13.wɑv
ffmpeg -y -i tts-poem-13.wɑv tts-poem-13.mp4

https://user-images.githubusercontent.com/77860779/171323687-d95b6cd3-19e8-4146-b634-7076869141ea.mp4




tts --text="kumɔ tsukinukɛ kɑzɛ kirisɑitɛ. kuɔn nɔ tɑjutɑi ɛ izɑnɑu." $enc --out_path tts-poem-14.wɑv
ffmpeg -y -i tts-poem-14.wɑv tts-poem-14.mp4



https://user-images.githubusercontent.com/77860779/171323703-e5e71be0-8232-430e-9a8f-526e7f30a2c7.mp4



#月には叢雲　華には風と
#たとえ愛の唄が永久　届かぬとも
#待ち続けよう　輪廻の時を
#慈しむ心に　誓って

tts --text="tsuki ni vɑ murɑkumɔ hɑnɑ ni vɑ kɑzɛ tɔ. tɑtɔɛ ɑi nɔ utɑ gɑ ɛikjuː tɔdɔkɑnu tɔmɔ." $enc --out_path tts-poem-15.wɑv
ffmpeg -y -i tts-poem-15.wɑv tts-poem-15.mp4



https://user-images.githubusercontent.com/77860779/171323717-dfb24ef1-bb48-4c2d-8360-492d14f3c6c4.mp4



tts --text="mɑtʃi tsuzukɛjɔu rinnɛ nɔ tɔki ɔ. itsukuʃimu kɔkɔrɔ ni tʃikɑttɛ." $enc --out_path tts-poem-16.wɑv
ffmpeg -y -i tts-poem-16.wɑv tts-poem-16.mp4


https://user-images.githubusercontent.com/77860779/171323719-cf79c97d-d750-473e-8b64-14767c8a2bd8.mp4

**Reply by kksteini** [reply]
いい声だ。ほんとに五日前より今は方がいいです。
I assume basic 5000 means you have that many sound files to train on?

**Reply by dmzubr** [reply]
Hi!

What was the step count of model that generate the samples above?

**Comment by neurlang** [comment]
yeah 5000 files, I forgot where I got the dataset from. I'm training on the nv3090ti branch. The most significant change is removal of this line, with the line I could not get the model to acceptably converge:

```
#self.nets.extend([DiscriminatorP(i, use_spectral_norm=use_spectral_norm) for i in periods])
```

Here is my config:
```
audio_config = BaseAudioConfig(
    sample_rate=48000,
    fft_size=1000,
    win_length=1000,
    hop_length=250,
    num_mels=80,
    preemphasis=0.0,
    ref_level_db=20,
    log_func="np.log",
    do_trim_silence=True,
    trim_db=45,
    mel_fmin=0,
    mel_fmax=8000,
    spec_gain=1.0,
    signal_norm=False,
    do_amp_to_db_linear=False,
)

config = VitsConfig(
    audio=audio_config,
    run_name="vits_ljspeech",
    batch_size=16,
    eval_batch_size=16,
    batch_group_size=5,
    num_loader_workers=8,
    num_eval_loader_workers=8,
    run_eval=True,
    save_step=1000,
    test_delay_epochs=-1,
    epochs=1000,
    text_cleaner="english_cleaners",
    use_phonemes=True,
    phoneme_language="multi-lingual",
    phonemizer="ipa_phonemizer",
    phoneme_cache_path=os.path.join(output_path, "phoneme_cache"),
    compute_input_seq_cache=True,
    print_step=25,
    print_eval=True,
    mixed_precision=True,
    output_path=output_path,
    datasets=[dataset_config],
)
```

**Comment by michaellin99999** [comment]
is this the kokoro dataset from kaggle?

**Reply by neurlang** [reply]
jsut-label: context labels and pronunciation data for JSUT corpus.

LICENCE:
   Text in  corpus is licensed as follows. The text data were modified and pronunciation information is added.
      basic5000
         - wikipedia (https://ja.wikipedia.org/) CC-BY-SA 3.0 
         - TANAKA corpus (http://www.edrdg.org/wiki/index.php/Tanaka_Corpus) CC-BY 2.0
         - JSUT (Japanese speech corpus of Saruwatari-lab., University of Tokyo) (https://sites.google.com/site/shinnosuketakamichi/publication/jsut). CC-BY-SA 4.0

**Comment by neurlang** [comment]
I've retrained the model on latest coqui tts:

#何を探してる？
tts/tts --text="nɑɲi vɔ sɑgɑʃi tɛru." $enc --out_path tts-poem-b-1.wav


https://user-images.githubusercontent.com/77860779/190903489-153eb132-c149-48f8-a998-ac47eda05f77.mp4



#まだ分からないけど
tts/tts --text="mɑdɑ vɑkɑrɑnɑikɛdɔ." $enc --out_path tts-poem-b-2.wav


https://user-images.githubusercontent.com/77860779/190903493-1575bfec-bf21-40e6-92d4-d922d8404a7b.mp4




#入り口はここかも (Hand in hand)
tts/tts --text="irigutʃi vɑ kɔkɔ kɑ mɔ, hɑnd in hɑnd." $enc --out_path tts-poem-b-3.wav



https://user-images.githubusercontent.com/77860779/190903496-f49839d5-f2ac-45e9-96ad-6d11860ba874.mp4



#それはキミの中に
tts/tts --text="sɔrɛ vɑ kimi nɔ nɑkɑ ɲi." $enc --out_path tts-poem-b-4.wav



https://user-images.githubusercontent.com/77860779/190903501-c8eb4fdf-86d0-4fe2-be63-133a56ff2f9a.mp4



#眠ってる情熱 もうすぐ目覚めそう
tts/tts --text="nɛmuttɛru dʒɔuɔnɛtsu mɔːuɔsugu mɛzɑmɛsɔː." $enc --out_path tts-poem-b-5.wav



https://user-images.githubusercontent.com/77860779/190903505-a1a4c8b8-cf8d-4fe2-97b3-ee016b0769d3.mp4



#気づいて！(Hand in Hand)
tts/tts --text="kidzuitɛ, hɑnd in hɑnd." $enc --out_path tts-poem-b-6.wav


https://user-images.githubusercontent.com/77860779/190903508-a5674325-3b69-48d4-94ba-88a7a66fe85c.mp4


#夢が生まれる 予感はただの
tts/tts --text="jumɛ gɑ humɑrɛru jɔkɑn vɑ tɑdɑ nɔ." $enc --out_path tts-poem-b-7.wav


https://user-images.githubusercontent.com/77860779/190903514-72b2d8ca-84e5-4611-8ef3-1ea0a4796e47.mp4


#錯覚じゃないはずさ (wow wo!)
tts/tts --text="sɑkkɑku dʒɑnɑi hɑzu sɑ, uɔu vɔ." $enc --out_path tts-poem-b-8.wav


https://user-images.githubusercontent.com/77860779/190903515-eb3fbbd4-5308-4c93-9192-e344b21c3959.mp4


#動きださなきゃはじまらないよ
tts/tts --text="ugɔkidɑsɑnɑkjɑ hɑdʒimɑrɑnɑi jɔ." $enc --out_path tts-poem-b-9.wav


https://user-images.githubusercontent.com/77860779/190903518-30d6c40d-0974-469b-95f1-a517c223aa42.mp4


#その手をつかまえたい
tts/tts --text="sɔnɔ tɛ vɔ tsukɑmɑɛtɑi." $enc --out_path tts-poem-b-10.wav


https://user-images.githubusercontent.com/77860779/190903524-32e303a7-483c-4227-97c3-04b72814322b.mp4


#かわれかわれって今日から
tts/tts --text="kɑ vɑrɛ kɑ vɑrɛ ttɛ kjɔː kɑrɑ." $enc --out_path tts-poem-b-11.wav


https://user-images.githubusercontent.com/77860779/190903526-d7700631-6dd5-4f76-a816-f58d7e0c1df7.mp4


#新しい世界へと
tts/tts --text="ɑtɑrɑʃiː sɛkɑi ɛ tɔ." $enc --out_path tts-poem-b-12.wav


https://user-images.githubusercontent.com/77860779/190903527-ed296077-f5e5-4af6-80f1-325e83e1610e.mp4


#かわれかわれって元気に
tts/tts --text="kɑvɑ rɛ kɑvɑ rɛ ttɛ gɛnki ɲi." $enc --out_path tts-poem-b-13.wav


https://user-images.githubusercontent.com/77860779/190903532-83d87a61-26eb-4af7-b33f-9b2a0086b820.mp4


#背中押してみよう
tts/tts --text="sɛnɑkɑɔ ʃitɛ mijɔu." $enc --out_path tts-poem-b-14.wav


https://user-images.githubusercontent.com/77860779/190903534-b5b45ab7-6f89-409f-9273-80b965c15236.mp4


#いっしょにだったら飛べるかな
tts/tts --text="isʃɔ ɲidɑttɑrɑ tɔbɛru kɑ nɑ." $enc --out_path tts-poem-b-15.wav


https://user-images.githubusercontent.com/77860779/190903536-5cc4c343-a2ab-4477-b9c2-4ca39cbf3194.mp4


#じゃあためらわないで もう諦めたくない
tts/tts --text="dʒɑː tɑmɛrɑvɑnɑidɛ mɔː ɑkirɑmɛtɑkunɑi." $enc --out_path tts-poem-b-16.wav


https://user-images.githubusercontent.com/77860779/190903543-c0c42ec8-30f0-4dd4-a0fc-451376afe9b8.mp4


#決めたよ！
tts/tts --text="kimɛtɑ jɔ." $enc --out_path tts-poem-b-17.wav


https://user-images.githubusercontent.com/77860779/190903545-9be190cb-d9cb-41c8-b8c9-b33ecb8d7401.mp4


#これからさあどこへ行こう？
tts/tts --text="kɔrɛkɑrɑ sɑː dɔkɔ ɛ ikɔu." $enc --out_path tts-poem-b-18.wav


https://user-images.githubusercontent.com/77860779/190903550-98729301-e5de-4528-aae5-420e9388e9c0.mp4


#手に手をとって行こう！
tts/tts --text="tɛ ɲi tɛ ɔ tɔttɛ ikɔu." $enc --out_path tts-poem-b-19.wav


https://user-images.githubusercontent.com/77860779/190903553-36ea21c5-4d7d-4f1a-bb38-eba5d761ab2e.mp4


#だれとめぐり逢う？
tts/tts --text="dɑrɛ tɔ mɛguri ɑu." $enc --out_path tts-poem-b-20.wav


https://user-images.githubusercontent.com/77860779/190903558-acab2b6d-4eba-4d9d-828f-615eec50b08f.mp4


#いまがその時と ピンときてないのかな？
tts/tts --text="imɑ gɑ sɔnɔtɔki tɔ pintɔ ki tɛnɑi nɔ kɑ nɑ." $enc --out_path tts-poem-b-21.wav


https://user-images.githubusercontent.com/77860779/190903561-3bceba71-3a40-47fd-ba22-df297a6cc32c.mp4


#(Hand in hand)
tts/tts --text="hɑnd in hɑnd." $enc --out_path tts-poem-b-22.wav

https://user-images.githubusercontent.com/77860779/190903567-2eb88636-9343-4c8a-a0d0-344b7a53f718.mp4



#直感を信じたいよ
tts/tts --text="tʃɔkkɑn ɔ ʃindʒitɑi jɔ." $enc --out_path tts-poem-b-23.wav


https://user-images.githubusercontent.com/77860779/190903573-020d57a3-3d89-4223-9db3-6b0b97f532cc.mp4


#心がふるえて 叫びたくなるんだ
tts/tts --text="kɔkɔrɔ gɑ furuɛtɛ sɑkɛbitɑku nɑru ndɑ." $enc --out_path tts-poem-b-24.wav


https://user-images.githubusercontent.com/77860779/190903577-b5784fe2-d93b-464d-93ed-cdadb552ae96.mp4


#夢をかたちにしたくなったら
tts/tts --text="jumɛ ɔ kɑtɑtʃi ɲi ʃitɑku nɑttɑrɑ." $enc --out_path tts-poem-b-25.wav


https://user-images.githubusercontent.com/77860779/190903582-7dc8677a-02f6-4ca6-84f1-7e63bd8d7908.mp4


#ひとりきりじゃなくて (wow wo!)
tts/tts --text="hitɔri kiri dʒɑnɑkutɛ, uɔu vɔ." $enc --out_path tts-poem-b-26.wav


https://user-images.githubusercontent.com/77860779/190903586-0c11c04f-9c00-4fdd-904c-dc8e24fbd57f.mp4


#動き出したい 同じ気持ちの
tts/tts --text="ugɔkidɑʃitɑi ɔnɑdʒi kimɔtʃi nɔ." $enc --out_path tts-poem-b-27.wav


https://user-images.githubusercontent.com/77860779/190903587-2e4d34e0-c68b-4edd-a000-62457f2209b5.mp4


#仲間がほしくなる
tts/tts --text="nɑkɑmɑ gɑ hɔʃiku nɑru." $enc --out_path tts-poem-b-28.wav


https://user-images.githubusercontent.com/77860779/190903589-f1216f15-bfca-4388-b16c-4eb2162d4868.mp4


#おいでおいでって無茶は
tts/tts --text="ɔidɛ ɔidɛ ttɛ mutʃɑ vɑ." $enc --out_path tts-poem-b-29.wav


https://user-images.githubusercontent.com/77860779/190903592-585852e2-604b-43b6-b3f1-3ae8d1b9e062.mp4


#承知で誘っちゃうよ
tts/tts --text="ʃɔːtʃi dɛ sɑsɔttʃɑu jɔ." $enc --out_path tts-poem-b-30.wav


https://user-images.githubusercontent.com/77860779/190903597-bc2db933-138a-49f9-9737-0f2d8ff4ecde.mp4


#おいでおいでって勇気を
tts/tts --text="ɔidɛ ɔidɛ ttɛ juːki ɔ." $enc --out_path tts-poem-b-31.wav


https://user-images.githubusercontent.com/77860779/190903602-78a96376-ff0f-48a7-914e-efb4e1140f3e.mp4


#だせばできるかも
tts/tts --text="dɑsɛbɑ dɛkiru kɑmɔ." $enc --out_path tts-poem-b-32.wav


https://user-images.githubusercontent.com/77860779/190903606-ec7f3f70-d20f-4edd-bb23-28288ef42c5b.mp4


#一緒にどっかへ飛びたいよ
tts/tts --text="isʃɔ ɲi, dɔkkɑ ɛ, tɔbitɑi jɔ." $enc --out_path tts-poem-b-33.wav


https://user-images.githubusercontent.com/77860779/190903610-f3d95b2b-6c75-435a-9c5d-de9cb9b0a786.mp4


#じゃあその手 ほら貸して
tts/tts --text="dʒɑː sɔnɔ tɛ, hɔrɑ kɑʃitɛ." $enc --out_path tts-poem-b-34.wav


https://user-images.githubusercontent.com/77860779/190903613-89eaa7f4-bc44-407c-a481-1448e4fe54e8.mp4


#ね 行くんだよ！
tts/tts --text="nɛ iku ndɑ jɔ." $enc --out_path tts-poem-b-35.wav


https://user-images.githubusercontent.com/77860779/190903617-2dc41cd3-1b2c-4b0c-94d9-d4e5318bdd7a.mp4


#夢が生まれる 予感はただの
tts/tts --text="jumɛ gɑ umɑrɛru jɔkɑn vɑ tɑdɑ nɔ." $enc --out_path tts-poem-b-36.wav


https://user-images.githubusercontent.com/77860779/190903622-daed61b5-c1be-44d6-bcc9-9114a0b73b59.mp4


#錯覚じゃないはずさ wow wo!
tts/tts --text="sɑkkɑku dʒɑnɑi hɑzu sɑ uɔu vɔ." $enc --out_path tts-poem-b-37.wav


https://user-images.githubusercontent.com/77860779/190903626-559dc4a3-f2c6-49bd-bc92-408866801f70.mp4


#かわれかわれって今日から
tts/tts --text="kɑvɑ rɛ kɑvɑ rɛ ttɛ kjɔː kɑrɑ." $enc --out_path tts-poem-b-38.wav


https://user-images.githubusercontent.com/77860779/190903630-d42551cd-ce8c-4067-a144-61d71759093f.mp4


#新しい世界へと
tts/tts --text="ɑtɑrɑʃiː sɛkɑi ɛ tɔ." $enc --out_path tts-poem-b-39.wav


https://user-images.githubusercontent.com/77860779/190903631-6173e458-62ab-44a4-83bc-340111ebe871.mp4


#かわれかわれって元気に
tts/tts --text="kɑvɑ rɛ kɑvɑ rɛ ttɛ gɛnki ɲi." $enc --out_path tts-poem-b-40.wav


https://user-images.githubusercontent.com/77860779/190903633-33aaeb19-1eaa-44ec-9536-97f163856bf4.mp4


#背中押してみよう
tts/tts --text="sɛnɑkɑɔ ʃitɛ mijɔu." $enc --out_path tts-poem-b-41.wav


https://user-images.githubusercontent.com/77860779/190903637-d916e22c-bbf3-4c12-aa27-277e15ffbabe.mp4


#おいでおいでって無茶は
tts/tts --text="ɔidɛ ɔidɛ ttɛ mutʃɑ vɑ." $enc --out_path tts-poem-b-42.wav


https://user-images.githubusercontent.com/77860779/190903639-969069cf-a1e6-4f20-be24-ee7d244e68bd.mp4


#承知で誘っちゃうよ
tts/tts --text="ʃɔːtʃi dɛ sɑsɔttʃɑu jɔ." $enc --out_path tts-poem-b-43.wav


https://user-images.githubusercontent.com/77860779/190903642-cf889069-579b-4e96-b635-5b95a42cc2a7.mp4


#おいでおいでって勇気を
tts/tts --text="ɔidɛ ɔidɛ ttɛ juːki ɔ." $enc --out_path tts-poem-b-44.wav


https://user-images.githubusercontent.com/77860779/190903644-47416d19-27e2-49a8-a1ad-a34b1bccb718.mp4


#だせばできるかも
tts/tts --text="dɑsɛbɑ dɛkiru kɑmɔ." $enc --out_path tts-poem-b-45.wav


https://user-images.githubusercontent.com/77860779/190903648-09024abf-5af2-485d-9fc4-dce920b3ca3a.mp4


#いっしょにだったら 飛べるかな
tts/tts --text="isʃɔ ɲidɑttɑrɑ tɔbɛru kɑ nɑ." $enc --out_path tts-poem-b-46.wav


https://user-images.githubusercontent.com/77860779/190903652-d2a2202a-6d1a-4b4f-af63-e1b54d6ea298.mp4


#じゃあためらわないで もう諦めたくない
tts/tts --text="dʒɑː tɑmɛrɑvɑnɑidɛ mɔː ɑkirɑmɛtɑkunɑi." $enc --out_path tts-poem-b-47.wav


https://user-images.githubusercontent.com/77860779/190903653-43a47cf4-66fc-4c79-a562-6ef2e3733933.mp4


#決めたよ！
tts/tts --text="kimɛtɑ jɔ." $enc --out_path tts-poem-b-48.wav


https://user-images.githubusercontent.com/77860779/190903654-817e5e96-8c67-47ff-b83f-0cc53734bbc3.mp4


#これからさあどこへ行こう？
tts/tts --text="kɔrɛkɑrɑ sɑː dɔkɔ ɛ ikɔu." $enc --out_path tts-poem-b-49.wav


https://user-images.githubusercontent.com/77860779/190903658-b3257f9d-09b2-43be-a642-d9a1d5db204c.mp4


#手に手をとって行こう！
tts/tts --text="tɛ ɲi tɛ ɔ tɔttɛ ikɔu." $enc --out_path tts-poem-b-50.wav


https://user-images.githubusercontent.com/77860779/190903660-73b7f3d2-a5fe-4f94-98fb-45d843c0f195.mp4

**Reply by neurlang** [reply]
Train time: 5hours
Step: 39000
![Screenshot from 2022-09-18 14-53-35](https://user-images.githubusercontent.com/77860779/190910491-1c9d44f3-5917-4779-a7e5-93e38412dcab.png)

**Reply by neurlang** [reply]
マッチ試験に備えて、本当に気合を入れて勉強しなきゃ。
Matchi shiken ni sonaete, hontōni kiai o irete benkyō shinakya.
mɑkki ʃikɛn ni sɔnɑɛtɛ, hɔntɔːni kiɑi ɔ irɛtɛ bɛnkjɔː ʃinɑkjɑ.


https://user-images.githubusercontent.com/77860779/190911549-fee56458-5d96-4ce3-9acf-82bb581d4fb7.mp4



カットの新決議案はかなりの衝撃を与える可能性もあります。
Katto no shin ketsugi-an wa kanari no shōgeki o ataeru kanōsei mo arimasu.
gɑttɔ nɔ ʃinkɛtsugi ɑn vɑ, kɑnɑri nɔ ʃɔːgɛki ɔ, ɑtɑɛru kɑnɔːsɛː gɑ ɑrimɑsu



https://user-images.githubusercontent.com/77860779/190911556-1f1957ef-0ea1-4831-b2cc-efe7335287db.mp4

**Reply by erogol** [reply]
Sounds really good! Congrats 🚀

Is this something we can open-source?

**Reply by freezerain** [reply]
Friend your graphs are so smooth, look at my VITS 😀😀😀
![image](https://user-images.githubusercontent.com/35431136/191035384-f13a3c3b-0501-4e00-af6b-99fc5ff83de2.png)

Anyway, even while I am not speaking japanese its clearly sounds better then your previous training and in general very pleasently sound model, congrats!

Can I ask you update on your config? 

I noticed you using english cleaner, isnt it destructive for non-english language? 

Also phonemizer="ipa_phonemizer"? Where did it come from? What phonemiser you actually use - ipa_phoneme or multi_phonemize.py (because you set  phoneme_language="multi-lingual")

I feel like my broken model confused by its characters/phonemizer

**Reply by neurlang** [reply]
@freezerain 

> Can I ask you update on your config?

```
diff train_vits.py ../../ljspeech/vits_tts/train_vits.py 
14c14
<     name="neurlang", meta_file_train="metadata.csv", path=os.path.join(output_path, "../Neurlang/")
---
>     name="ljspeech", meta_file_train="metadata.csv", path=os.path.join(output_path, "../LJSpeech-1.1/")
17c17
<     sample_rate=48000, win_length=1024, hop_length=256, num_mels=80, mel_fmin=0, mel_fmax=None
---
>     sample_rate=22050, win_length=1024, hop_length=256, num_mels=80, mel_fmin=0, mel_fmax=None
22,23c22,23
<     run_name="vits_neurlang",
<     batch_size=16,
---
>     run_name="vits_ljspeech",
>     batch_size=32,
29d28
<     save_step=1000,
34,35c33
<     phoneme_language="multi-lingual",
<     phonemizer="ipa_phonemizer",
---
>     phoneme_language="en-us",
```

@freezerain 

> I noticed you using english cleaner, isnt it destructive for non-english language?

I don't think the cleaner cleans anything. My dataset is already clean.

@freezerain 

> Where did it come from? What phonemiser you actually use

I don't use a phonemiser currently. I've transcribed the dataset to IPA and I train on that. When I'm invoking the model, I give it IPA input.


> Sounds really good! Congrats rocket
> 
> Is this something we can open-source?

@erogol , I would be glad to open source it, but I don't have a phonemizer. I need to find a good neural phonemizer and train it on my dataset. My dataset transcript is currently IPA, and I train on that.

I also have the original Japanese transcript for my dataset, so it's just missing the phonemizer.

**Reply by erogol** [reply]
There is one neural phonemizer here. https://github.com/as-ideas/DeepPhonemizer

If it works for you we can consider integrate it to 🐸TTS too

**Reply by mthrok** [reply]
What are "マッチ試験" and "カットの新決議案" supposed to mean?

**Reply by neurlang** [reply]
> There is one neural phonemizer here. https://github.com/as-ideas/DeepPhonemizer
> 
> If it works for you we can consider integrate it to frogTTS too

This is a good Idea. But to train the phonemizer, I don't have word-word pairs (only sentence-sentence currently).
It would take a lot of effort to align individual JP words to their ipa versions.

If anyone knows a neural phonemizer that can train on whole sentences that would be great.

**Reply by neurlang** [reply]
> What are "マッチ試験" and "カットの新決議案" supposed to mean?

Even though this is my best model, I don't actually read japanese. Sorry. But it's from the dataset.

**Reply by kdrkdrkdr** [reply]
Could you share your train_vits.py file?
It doesn't work with the options you posted.
